src/dev/build/tasks/os_packages/docker_generator/resources/bin/kibana-docker

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
+
+# Run Kibana, using environment variables to set longopts defining Kibana's
+# configuration.
+#
+# eg. Setting the environment variable:
+#
+#       ELASTICSEARCH_STARTUPTIMEOUT=60
+#
+# will cause Kibana to be invoked with:
+#
+#       --elasticsearch.startupTimeout=60
+
+kibana_vars=(
```

Comment:
@jarpy is there a reason we can't take the same approach as ES is with matching environment variables, except for ours would match uppercase?

https://github.com/elastic/elasticsearch/blob/master/distribution/docker/src/docker/bin/docker-entrypoint.sh#L51-L62

Reply:
In the meantime, @mistic would you mind updating this list as it's recently been updated in the kibana-docker repo: https://github.com/elastic/kibana-docker/pull/128

Reply:
@tylersmalley done

Reply:
You could do that, but:

You'll have to do something for flags that don't have dots, like `regionmap`.

You'll have to ask the users to switch to mixed-case env vars. Unlike ES, Kibana has mixed-case flags, so it essentially _encodes information in character case_. You can't smash case to upper or lower without losing that information. That's why this horrible "translation table" exists. Arguably, we should have just asked for mixed-case, 1:1 mapped env vars to begin with. That might be a good change for 7.x.

Reply:
Note that there was a time when we couldn't use dots in env vars, but that issue is now resolved. It does, however, explain why we also `s/_/./`